app/src/main/java/com/sdp/swiftwallet/presentation/cryptoGraph/CryptoGraphActivity.java

```diff
@@ -242,6 +246,51 @@ private void onItemSelectedHandler(AdapterView<?> adapterView, View view, int po
             .show();
     }
 
+    /**
+     * Sets the spinner for the currency to show
+     */
+    private void setCurrencyToShowSpinner() {
+        this.currencyToShowspinner = findViewById(R.id.idCurrencyToShow);
+
+        ArrayAdapter<CharSequence> adapter = ArrayAdapter
+                .createFromResource(this, R.array.main_crypto_names,
+                        android.R.layout.simple_spinner_item);
+        adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+
+        this.currencyToShowspinner.setAdapter(adapter);
+        // Display the interval or does nothing if not selected
+        this.currencyToShowspinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> adapterView, View view, int i, long l) {
+                if(i > 0) {
+                    onItemSelectedCurrencyHandler(adapterView, view, i, l);
+                    String newCurrency = (String) adapterView.getItemAtPosition(i);
+                    rateSymbol = newCurrency + "USDT";
+                    TextView textView = findViewById(R.id.idCurrencyToShowName);
+                    textView.setText(newCurrency);
+                    candleStickChart = getData();
+                }
+            }
+
+            @Override
+            public void onNothingSelected(AdapterView<?> adapterView) {
+            }
+        });
+    }
+
+    /**
+     * Handler for the currency to show adapter, display it when selected
+     */
+    private void onItemSelectedCurrencyHandler(AdapterView<?> adapterView, View view, int position,
+                                       long id) {
+        Adapter adapter = adapterView.getAdapter();
+        String symbol = (String) adapter.getItem(position);
+
+        Toast
+                .makeText(getApplicationContext(), "Selected Currency : " + symbol+"USDT", Toast.LENGTH_SHORT)
```

Comment:
For displaying a simple toast we made a function in HelperFunction which takes less space if you want to use it later